assets/js/modules/adsense/datastore/base.js

```diff
@@ -39,9 +38,6 @@ const baseModuleStore = Modules.createModuleStore( 'adsense', {
 		'webStoriesAdUnit',
 		'autoAdsDisabled',
 	],
-	adminPage: isFeatureEnabled( 'unifiedDashboard' )
```

Comment:
So, we don't use this `adminPage` property anymore in our modules, therefore we can remove it from `createInfoStore` and `createModuleStore` factories respectfully and use just `googlesitekit-dashboard` in the `getAdminScreenURL` selector.

Reply:
Sounds good @eugene-manuilov . Updated!